XenAdmin/TabPages/PhysicalStoragePage.cs

```diff
@@ -60,9 +61,7 @@ public PhysicalStoragePage()
         {
             InitializeComponent();
 
-            listViewSrs.SmallImageList = Images.ImageList16;
-            listViewSrs.ListViewItemSorter = lvwColumnSorter;
-            listViewSrs.SelectedIndexChanged += new EventHandler(listViewSrs_SelectedIndexChanged);
+            dataGridViewSr.SelectionChanged += new EventHandler(dataGridViewSrs_SelectedIndexChanged);

```

Comment:
Event registration can be done without explicitly specifying the handler: `dataGridViewSr.SelectionChanged += dataGridViewSrs_SelectedIndexChanged;`

Reply:
Do you mean you prefer to register handler in *Designer*?

Reply:
You can register it in the designer - in which case visual studio will add the event handler automatically and you don't need to worry about anything. But my comment was only about style, there isn't anything wrong if you register it as you have, it was only a sugegstion for shorter and more readable code.